packages/Search/test/tstDetailsTreeTraversal.cpp

```diff
@@ -21,19 +21,21 @@
 
 #include <random>
 
-#include "DTK_EnableViewComparison.hpp"
+#include "ArborX_EnableViewComparison.hpp"
 
 #include <boost/test/unit_test.hpp>
 
-namespace dtk = DataTransferKit::Details;
+#define BOOST_TEST_MODULE DetailsTreeTraversal
+
+namespace details = ArborX::Details;
```

Comment:
How about a using-directive?

Reply:
Out of scope of this PR, can be addressed later.